main.yml

```
@@ -11,6 +11,18 @@ access-requests:
     paths:
       - /internal/access-requests
 
+acs:
+  title: Advanced Cluster Security
+  api:
+    version:
+      - v1
+  description: Frontend for Red Hat employees to manage ACS instances
+  deployment_repo: https://github.com/stackrox/acs-ui-build
+  frontend:
+    paths:
+      - /acs
```

This should include the full path
```suggestion
      - /application-services/acs
```